csharp/src/Ice/Proxy.cs

```diff
@@ -376,7 +376,7 @@ public static class Proxy
             {
                 Reference reference = proxy.IceReference;
 
-                IReadOnlyDictionary<string, string>? context = request.Context ?? reference.CurrentContext();
+                IReadOnlyDictionary<string, string> context = request.Context;
```

Comment:
We don't need this temporary variable.

Reply:
I didn't mean to add this fix (see my blog post about it). But since this didn't raise any eyebrows I'll commit it and get rid of the temp.